commons/commons-jdbc/src/main/java/com/thinkbiganalytics/jdbc/util/DatabaseType.java

```diff
@@ -39,6 +39,7 @@
  */
 public enum DatabaseType {
     // For the preview pattern: {0}=columns, {1}=schema, {2}=table, {3}=limit
+    IMPALA("Impala", "jdbc:impala:", "select 1", "SELECT {0} FROM {1}.{2} LIMIT {3}"),
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "select 1" 10 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "SELECT {0} FROM {1}.{2} LIMIT {3}" 8 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)